src/main/java/beans/StatusResponseBean.java

```diff
@@ -0,0 +1,41 @@
+package beans;
+
+/**
+ * A generic status response that returns either "ok" or "error" with a message
+ * if it has errored.
+ */
+public class StatusResponseBean {
```

Comment:
This class seems to have a lot in common with NotificationMessageBean - they should probably be merged